pkg/types/nutanix/helpers.go

```diff
@@ -178,3 +178,8 @@ func getTaskStatus(clientV3 nutanixclientv3.Service, taskUUID string) (string, e
 func RHCOSImageName(infraID string) string {
 	return fmt.Sprintf("%s-rhcos", infraID)
 }
+
+// Ptr takes a pointer to the passed object.
+func Ptr[T any](v T) *T {
```

Comment:
Not sure why we need this when this is already vendored 
https://pkg.go.dev/k8s.io/utils/pointer
or
https://pkg.go.dev/k8s.io/utils/ptr

Reply:
Removed this one.